cfn-templates/cid-cfn.yml

```diff
@@ -4,28 +4,26 @@ Metadata:
   AWS::CloudFormation::Interface:
     ParameterGroups:
       - Label:
-          default: 'Prerequisites'
+          default: 'Common'
         Parameters:
           - PrerequisitesQuickSight
-          - PrerequisitesCUR
           - PrerequisitesQuickSightPermissions
-      - Label:
-          default: 'Mandatory parameters'
-        Parameters:
           - QuickSightUser
-          - CURBucketPath
-          - CURIsManagedByCloudFormation
       - Label:
-          default: Available dashboards. Keep all 'no' for the first deployment. Go to QuickSight, change Permission and after that update the stack putting to 'yes' 3 first dashboards.
+          default: CUDOS Cost-Intelligence-Dashboard and KPI-Dashboard. Require deployment of CUR via CloudFormation (cur-aggregation.yaml) or manually 24h before.
         Parameters:
+          - CURBucketPath
           - DeployCUDOSDashboard
           - DeployCostIntelligenceDashboard
           - DeployKPIDashboard
+      - Label:
+          default: Trusted Advisor and Compute Optimiser Dashboards. Require deployment of Optimization Data Collection Lab (https://wellarchitectedlabs.com/cost/300_labs/300_optimization_data_collection/)
+        Parameters:
+          - OptimizationDataCollectionBucketPath
           - DeployTAODashboard
           - DeployComputeOptimizerDashboard
-          - CostOptimizationDataCollectionBucketPath
       - Label:
-          default: 'Optional Parameters (do not change unless you know why)'
+          default: 'Technical Parameters. Please do not change.'
```

Comment:
Mayebe 'Optional Technical Parameters. Only required in rare cases of customized deployments. Please do not change'?

Reply:
Customers will wonder why you showing them this then, so we can answer this by providing more precise wording, required in rare cases of not standard deployment. We can use not standard instead of customized

Reply:
We explain that these are technical parameters.

Reply:
ok